src/app/bav/controllers/landingPage.js

```diff
@@ -1,5 +1,11 @@
 const BaseController = require("hmpo-form-wizard").Controller;
 
-class LandingPageController extends BaseController {}
+class LandingPageController extends BaseController {
+  locals(req, res) {
+    super.locals(req, res, () => {
+      req.sessionModel.set("isLanding", true);
```

Comment:
Shall we add a unit test for this?

Reply:
Yes and should I add a complementary one for the inverse in confirmDetails.js?

Reply:
That would be great!